closed/GensrcJ9JCL.gmk

```diff
@@ -48,6 +48,11 @@ $(eval $(call SetupCopyFiles,COPY_OVERLAY_FILES, \
 		src/java.base/share/classes/sun/security/jca/ProviderConfig.java \
 		src/java.base/share/classes/sun/security/jca/ProviderList.java \
 		src/java.base/unix/classes/java/lang/ProcessEnvironment.java \
+		src/java.se/share/data/jdwp/jdwp.spec \
+		src/jdk.jdi/share/classes/com/sun/tools/example/debug/tty/EventHandler.java \
+		src/jdk.jdi/share/classes/com/sun/tools/example/debug/tty/EventNotifier.java \
+		src/jdk.jdi/share/classes/com/sun/tools/example/debug/tty/TTY.java \
```

Comment:
Missed `TTYResources.java`.

Reply:
Added in https://github.com/ibmruntimes/openj9-openjdk-jdk/commit/4e515e9b0139d5148fe5e1599ba67428aa112c0e